src/main/java/BasicMAPF/Solvers/PIBT/PIBT_Solver.java

```diff
@@ -412,7 +478,9 @@ private Solution makeSolution() {
             if (lastChangedLocationTime < plan.getEndTime() && lastChangedLocationTime > 0) {
                 SingleAgentPlan trimmedPlan = new SingleAgentPlan(agent);
                 for (int t = 1; t <= lastChangedLocationTime; t++) {
-                    trimmedPlan.addMove(plan.moveAt(t));
+                    if (plan.moveAt(t) != null) {
+                        trimmedPlan.addMove(plan.moveAt(t));
+                    }
```

Comment:
Isn't this condition always true?

Reply:
In offline its always true, but when I run this code in lifelong some scenarios caused an error in line 482 since plan is null